src/Form/AutologoutSettingsForm.php

```diff
@@ -147,14 +147,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       $form['table'][] = array(
         'autologout_role_' . $key => array(
           '#type' => 'checkbox',
-          '#default_value' => $config->get('role_' . $key),
+          '#default_value' => $config->get('autologout_role_' . $key),
```

Comment:
This change seems to be redundant. See the commit : https://www.drupal.org/commitlog/commit/1626/bc515b847883b3bcd0cbbf4912265955fe55d002